<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="backgroundPhoto">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>650</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Media/Des3.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="predictionText">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>510</y>
     <width>341</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>22</pointsize>
     <weight>8</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 70 22pt &quot;Century Gothic&quot;; 
color: black;</string>
   </property>
   <property name="text">
    <string>Predicted action:</string>
   </property>
  </widget>
  <widget class="QLabel" name="trueText">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>570</y>
     <width>341</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>22</pointsize>
     <weight>8</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 70 22pt &quot;Century Gothic&quot;; 
color: black;
</string>
   </property>
   <property name="text">
    <string>True action:</string>
   </property>
  </widget>
  <widget class="QLabel" name="image">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>640</width>
     <height>360</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>640</width>
     <height>360</height>
    </size>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="FACButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>11</pointsize>
     <weight>8</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-radius: 20px;
	background-color: rgb(224, 224, 224);
	font: 70 11pt &quot;Century Gothic&quot;; 
   color: black;
}

QPushButton:hover{
	background-color: rgb(73, 96, 96);
}
QPushButton:pressed{
	background-color: black;
}</string>
   </property>
   <property name="text">
    <string>Go to main page</string>
   </property>
  </widget>
  <widget class="QPushButton" name="previousButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>560</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>14</pointsize>
     <weight>8</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-radius: 20px;
	background-color: rgb(224, 224, 224);
	font: 70 14pt &quot;Century Gothic&quot;; 
   color: black;
}

QPushButton:hover{
	background-color: rgb(73, 96, 96);
}
QPushButton:pressed{
	background-color: black;
}</string>
   </property>
   <property name="text">
    <string>Previous Frame</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextButton">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>560</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>14</pointsize>
     <weight>8</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-radius: 20px;
	background-color: rgb(224, 224, 224);
	font: 70 14pt &quot;Century Gothic&quot;; 
   color: black;
}

QPushButton:hover{
	background-color: rgb(73, 96, 96);
}
QPushButton:pressed{
	background-color: black;
}</string>
   </property>
   <property name="text">
    <string>Next Frame</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
